Cron/KlSyncs.php

```diff
@@ -129,11 +129,19 @@ private function sendUpdatesToApp(array $groupedRows, bool $isRetry = false)
                     $eventTime = $decodedPayload['time'];
                     unset($decodedPayload['time']);
 
+                    $storeId = '';
+                    if (isset($decodedPayload['StoreId']))
```

Comment:
Im just curious is there any possible scenario that StoreId wouldn't be set?

Reply:
In case this change is made live and some rows have been created before the change was deployed, and a few rows are pending sync to Klaviyo, these won't have the storeId key in the payload

Reply:
I dunno if its worth putting a comment or TODO that its only there for backwards compatibility and maybe in the future we just remove it?

Reply:
Yup, good thought, I'll add a todo